src/components/pages/summit-2024/top-banner/top-banner.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import Link from 'components/shared/link';
+
+const TopBanner = () => (
+  <div className="relative flex h-[48px] flex-row items-center justify-center space-x-6 overflow-hidden bg-black py-2 sm:h-auto sm:flex-col sm:space-x-0 sm:space-y-1 sm:pb-5">
+    <div className="flex flex-row items-center">
+      <span className="font-sans text-lg font-medium leading-none text-white md:text-base">
+        CFPs opens from 5th to 17th July
```

Comment:
```suggestion
        CFPs open until 17th July
```

Reply:
I don't think there's any reason not to open the CFP as soon as the web site is up, right @xmulligan ?

Reply:
yes, that makes sense to me too